test/resolvers/project.js

```diff
@@ -743,4 +743,57 @@ describe('Project resolver', () => {
     });
   });
 
+  describe('update-issue-date', () => {
+    beforeEach(() => {
+      const originalIssueDate = new Date('2020-01-17').toISOString();
+      const originalExpiryDate = new Date('2025-01-17').toISOString();
+      const duration = { years: 5, months: 0 };
+
+      return Promise.resolve()
+        .then(() => this.models.Project.query().insert([
+          {
+            id: projectId,
+            status: 'active',
+            title: 'Active project wrong issue date',
+            issueDate: originalIssueDate,
+            expiryDate: originalExpiryDate,
+            establishmentId: 8201,
+            licenceHolderId: profileId
+          }
+        ]))
+        .then(() => this.models.ProjectVersion.query().insert([
+          {
+            id: '574266e5-ef34-4e34-bf75-7b6201357e75',
+            projectId,
+            data: {
+              duration
+            },
+            status: 'granted',
+            createdAt: originalIssueDate
+          }
+        ]));
+    });
+
+    it('can change the issue date of a project', () => {
+      const newIssueDate = new Date('2018-08-15').toISOString();
+      const expectedExpiryDate = moment(newIssueDate).add(5, 'years').toISOString();
```

Comment:
Does this work if you hard code the expected date as `2023-08-15`?

I ask because the code does a `moment().add()` and then the test does a `moment).add()` in order to vlaidate that it worked, which is a little circular.

Reply:
I thought `Date()` added the current time if you didn't specify it, and that I'd have to then worry about millisecond differences, but it only adds it if you don't specify any param at all.